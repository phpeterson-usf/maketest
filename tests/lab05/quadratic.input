$(repo_dir)/quadratic_main
