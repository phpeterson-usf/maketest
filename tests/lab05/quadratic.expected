quadtratic_c(1, 1, 1, 1) = 3
quadtratic_s(1, 1, 1, 1) = 3
quadtratic_c(2, 2, 3, 4) = 18
quadtratic_s(2, 2, 3, 4) = 18
